config/opensearch.yml

```diff
@@ -0,0 +1,27 @@
+cluster.name: okpt-cluster
+network.host: 0.0.0.0
+discovery.type: single-node
+plugins.security.disabled: true
+# ######## Start OpenSearch Security Demo Configuration ########
```

Comment:
Any reason for including all of this commented out code?

Reply:
The security settings seemed finicky so I thought it may be useful for others though I don't see myself using them. Should I remove the commented out settings?

Reply:
No I think you can leave them in.